tasks.py

```diff
@@ -208,60 +207,6 @@ def release(ctx, function=None, description=None):
         __update_release_alias(ctx, func, new_version, description)
 
 
-@task
```

Comment:
With the current number of recordings we have in our account these days it's so slow it's basically unusable and it would require making an API request repeatedly that is likely to disrupt other services.